### PR TITLE
added support for symfony 3.0

### PR DESCRIPTION
**Pull request by smoench**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,13 +7,11 @@ php:
   - 5.5
   - 5.6
   - 7
-  - nightly
   - hhvm
 
 matrix:
   allow_failures:
     - php: 7
```
Comment (tolry) [inline .travis.yml]: Maybe remove this allowed failure while we're at it? :smile:

**Comment by DavidBadura** [comment]
:+1:

**Comment by tolry** [comment]
@smoench could you rebase, pls?

**Comment by DavidBadura** [comment]
@simpspector/owners ping

**Comment by smoench** [comment]
@DavidBadura why are you so noisy? this PR is still marked as WIP!

**Comment by DavidBadura** [comment]
sorry, i haven't seen the label :-/

**Comment by smoench** [comment]
@DavidBadura so, you are going to fix the master and release a new version simpspector/twig-lint, it is using the deleted `smfony-3.0`!?

**Comment by larsborn** [comment]
me neither :-o

**Pull request merged**